mobile/library/common/jni/jni_interface.cc

```diff
@@ -1125,6 +1127,17 @@ void setString(JNIEnv* env, jstring java_string, EngineBuilder* builder,
   }
 }
 
+// Convert jstring to std::string
+std::string getCppString(JNIEnv* env, jstring java_string) {
+  if (!java_string) {
+    return "";
+  }
+  const char* native_java_string = env->GetStringUTFChars(java_string, nullptr);
+  std::string cpp_string = std::string(native_java_string);
+  env->ReleaseStringUTFChars(java_string, native_java_string);
+  return cpp_string;
+}
```

Comment:
We currently only have a helper function `setString` which works for builder APIs which set exactly one string.  This approach works for any APIs that need cpp strings